### PR TITLE
Enable TOC rendering for all builds

### PR DESCRIPTION
**Pull request by Lennonka**
Cherry-pick into:

* [x] Foreman 3.1

### CODE REVIEW

**Comment by melcorr** [comment]
I'll test this.

File: guides/doc-Content_Management_Guide/master.adoc [hunk 1]
```diff
@@ -1,10 +1,8 @@
 = Content Management Guide
```
Comment (ekohl) [inline guides/doc-Content_Management_Guide/master.adoc:1]: Any idea why this isn't using `guides/common/header.adoc`?

**Review by ekohl — CHANGES_REQUESTED** [review]
I'd prefer to modify the common header:
https://github.com/theforeman/foreman-documentation/blob/28f70052552b106545f33b69625931fc516f2051/guides/common/header.adoc#L3-L7

If you change that, it should be a much smaller change.

**Comment by Lennonka** [comment]
Great points, thanks, I'll take care of it.

**Comment by Lennonka** [comment]
I did a proper cleanup of the headers of all guides. Hopefully the builds pass!

**Comment by Lennonka** [comment]
@melcorr @ekohl Please, review and test. Builds are passing.

**Review by ekohl — APPROVED** [review]
Aligning things makes me very happy. I also love that this patch is a net negative in number of lines without sacrificing anything.

@melcorr feel free to give it a final review and merge it when you feel it's ready.

**Review by melcorr — APPROVED** [review]
I built the guides and everything looks good here. 
Thank you for your work on this @Lennonka

**Comment by melcorr** [comment]
Cherry-picked so the TOC is available for any link sharing for beta Satellite 7 work if needed 

   e94f7a4e..63076c12  3.1 -> 3.1

**Pull request merged**